### PR TITLE
[Dropzone] Fix wonky border radius

### PR DESCRIPTION
**Pull request by martenbjork**
The border radius of the `DropZone` component was messed up. This PR fixes that.

dropzone-border-radius

### 🎩 checklist

- [x] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [x] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [x] Updated the component's `README.md` with documentation changes
- [x] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

File: .changeset/breezy-mirrors-fetch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Fixed the border radius of the DropZone component.
```
Comment (chloerice) [inline .changeset/breezy-mirrors-fetch.md]:
```suggestion
Fixed the border radius of `DropZone`
```

File: polaris-react/src/components/DropZone/DropZone.scss [hunk 2]
```diff
@@ -125,7 +125,7 @@ $dropzone-stacking-order: (
 
 .Container {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  @include focus-ring($border-width: 1px);
+  @include focus-ring($size: 'wide', $border-width: 1px);
```
Comment (chloerice) [inline polaris-react/src/components/DropZone/DropZone.scss]:
```suggestion
  @include focus-ring($size: 'wide', $border-width: var(--p-border-width-1));
```

**Review by chloerice — APPROVED** [review]
🙌🏽 🚢

**Review by kyledurand — APPROVED** [review]

**Comment by kyledurand** [comment]
Chromatic seems to be having a tough day. Manually 🎩 'd and looks much better 👍

File: polaris-react/src/components/DropZone/DropZone.scss (hunk 2 above)
Comment (martenbjork) [reply polaris-react/src/components/DropZone/DropZone.scss]: Awesome 🦅👀🧠

**Pull request merged**